internal/utils/csi.go

```diff
@@ -331,10 +366,15 @@ var TopologyContainerArg = "--feature-gates=Topology=true"
 var RecoverVolumeExpansionFailureContainerArg = "--feature-gates=RecoverVolumeExpansionFailure=true"
 var EnableVolumeGroupSnapshotsContainerArg = "--enable-volume-group-snapshots=true"
 var ForceCephKernelClientContainerArg = "--forcecephkernelclient=true"
+var LogToFile = "--logtostderr=false"
+var LogToStderr = "--alsologtostderr=true"
```

Comment:
Argument const names should match the argument string and should always end in `ContainerArg`. This is done for each of readability when looking at the definition of the container arguments. 

```
var LogToStdErrContainerArg = "--logtostderr=false"
var AlsoLogToStdErrContainerArg = "--alsologtostderr=true"
```